src/geo/leaflet/leaflet-cartodb-layer-group-view.js

```diff
@@ -104,17 +104,30 @@ var LeafletCartoDBLayerGroupView = L.TileLayer.extend({
   },
 
   featureOver: function (e, latlon, pixelPos, data, layer) {
-    // dont pass leaflet lat/lon
-    this.trigger('featureOver', e, [latlon.lat, latlon.lng], pixelPos, data, layer);
+    this.trigger('featureOver', {
+      layer: this.model.getLayerAt(layer),
+      layerIndex: layer,
+      latlng: [latlon.lat, latlon.lng],
```

Comment:
This would also be nice, but we can tackle it in a separate PR as it'd require many additional changes.